src/library_async.js

```diff
@@ -60,22 +76,51 @@ mergeInto(LibraryManager.library, {
       return id;
     },
 
-#if ASSERTIONS
     instrumentWasmImports: function(imports) {
+#if ASYNCIFY_DEBUG
+      err('asyncify instrumenting imports');
+#endif
+#if ASYNCIFY == 2
+      // TODO we could perhaps add an init function and put this there, but
+      //      this should work for now.
+      Asyncify.suspender = new WebAssembly.Suspender();
```

Comment:
You can't do `$suspender: "new WebAssembly.Suspender()"` as the top level of the library object?